specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/putComputeOneSkuQuotaRequest.json

```diff
@@ -0,0 +1,53 @@
+{
+  "parameters": {
+    "scope": "subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus",
+    "resourceName": "standardFSv2Family",
+    "api-version": "2021-03-15",
+    "createQuotaRequest": {
+      "properties": {
+        "limit": 200,
+        "unit": "Count",
+        "name": {
+          "value": "standardFSv2Family"
+        }
+      }
+    }
+  },
+  "responses": {
+    "201": {
+      "body": {
+        "id": "/subscriptions/D7EC67B3-7657-4966-BFFC-41EFD36BAAB3/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/serviceLimitsRequests/2B5C8515-37D8-4B6A-879B-CD641A2CF605",
```

Comment:
>2B5C8515-37D8-4B6A-879B-CD641A2CF605 [](http://example.com/codeflow?start=160&length=36)

The resource name should match the request parameter 'standardFSv2Family'

Reply:
I will update it to match the resource name.

Reply:
Updated the examples to match the SKU name.